Fix autoreconf with autoconf 1.12+ by adding AM_PROG_AR
https://code.google.com/p/libkml/issues/detail?id=186
Fix autoreconf with autoconf 1.14+ by adding subdir-objects
https://code.google.com/p/libkml/issues/detail?id=190
--- configure.ac.orig	2010-01-13 18:16:35.000000000 -0600
+++ configure.ac	2015-05-10 08:16:51.000000000 -0500
@@ -6,7 +6,8 @@
 AC_INIT(libkml, 1.2.0, http://code.google.com/p/libkml/issues)
 AC_CONFIG_SRCDIR(README)
 AC_CONFIG_AUX_DIR(config)
-AM_INIT_AUTOMAKE(-Wall -Werror)
+AM_INIT_AUTOMAKE(-Wall -Werror subdir-objects)
+m4_ifdef([AM_PROG_AR], [AM_PROG_AR])
 
 AC_PROG_CPP
 AC_PROG_CXX
